src/fsharp/TypedTreeOps.fs

```diff
@@ -2795,6 +2796,7 @@ type DisplayEnv =
         useColonForReturnType = false
         shrinkOverloads = true
         printVerboseSignatures = false
+        escapeKeywordNames = false
```

Comment:
I think we shouldn't even have `escapeKeywordNames`. We should always put the "``" around names that are keywords.

Reply:
That will cause things like error messages to show an escaped name for a parameter. That's what led to this change, since it caused a test to fail where the error message was changed

Reply:
Interesting, that makes sense.